util/testUtils.js

```diff
@@ -22,3 +25,30 @@ export const middlewareDispatcher = middleware => (storeData, action) => {
 	return dispatched;
 };
 
+export const parseCookieHeader = (cookieHeader) => {
```

Comment:
made the update for this file, although I think in general there are good reasons to use different function declaration syntaxes in different cases. Arrow functions are really nice for currying (`const fn = x => y => z => { ... }`) and when you return immediately (`const fn = x => Object.keys(x).map(key => x[key])`), but I tend to have a slight preference for `function fn(x)` when you have a beefy function block.

This is all mostly a mild aesthetic pref, though, so I'm happy to adjust wherever